diagnostic.md

```diff
@@ -4,7 +4,10 @@ Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
 <!-- your answer starts here -->
-
+document.querySelectorAll( "h1" );
+document.querySelectorAll( "ol" );
+document.querySelectorAll( "li" );
+document.querySelectorAll( "form" );
```

Comment:
I think you misunderstood the question, although what you have here is not technically incorrect. What we were looking for were 4 examples of selectors that one might use if you were working with this particular page. Some of these are `header header-logged-in true`, and `text-gray`. Can you see where those could be found?